test/environments/EnvironmentSearch.test.tsx

```diff
@@ -1,13 +1,19 @@
 import React from "react";
+import { Provider } from "react-redux";
 import { fireEvent, render, RenderResult } from "@testing-library/react";
 import { EnvironmentsSearch } from "../../src/features/environments";
+import { store } from "../../src/store";
 
 describe("<EnvironmentsSearch />", () => {
   const mockOnChange = jest.fn();
   let component: RenderResult;
 
   beforeEach(() => {
-    component = render(<EnvironmentsSearch onChange={mockOnChange} />);
+    component = render(
+      <Provider store={store}>
+        <EnvironmentsSearch onChange={mockOnChange} />
+      </Provider>
+    );
```

Comment:
Nit: The test passes with this. But ideally, this would be wrapped into `mockTheme` as well, like the other tests, right?